apps/portal/controllers/apis/login.jag

```diff
@@ -73,6 +74,21 @@ var user;
                     return;
                 }
                 session.put("Loged", "true");
+
+                var ApiUrl = 'https://localhost:9443/notification/services/notifications/notificationApi/notifications/login'+'?tenantId=' + user.tenantId;
```

Comment:
The url cannot be hard coded, especially port and host name. If needs to be taken from the config file.